libyul/AsmAnalysis.cpp

```diff
@@ -585,7 +585,7 @@ bool AsmAnalyzer::warnOnInstructions(evmasm::Instruction _instr, SourceLocation
 	else if (_instr == evmasm::Instruction::PC)
 	{
 		m_errorReporter.error(
-			4316_error,
+			6719_error,
```

Comment:
Why has this changed?

Reply:
The error_id script.

Reply:
Maybe it is better to fix this in develop and merge it to breaking.